plugins/gatsby-theme-iterative-docs/gatsby-node.js

```diff
@@ -33,6 +34,25 @@ exports.createSchemaCustomization = async api => {
   ])
 }
 
+exports.onCreateWebpackConfig = ({ actions }) => {
+  actions.setWebpackConfig({
+    resolve: {
+      alias: {
+        [path.resolve(__dirname, 'sidebar')]: path.resolve(
+          'src',
+          'gatsby-theme-iterative-docs',
+          'sidebar'
+        ),
+        [path.resolve(__dirname, 'redirects')]: path.resolve(
+          'src',
+          'gatsby-theme-iterative-docs',
+          'redirects'
+        )
+      }
+    }
+  })
+}
```

Comment:
The meat and potatoes to getting sidebar resolution working: this is similar to theme shadowing within Gatsby and uses the same directory structure on the consumer end, but the actual node files in the same location allow `gatsby-node` and other pure node modules to work with the same imports, and so allowing the modules that import these to be used in both environments, enabling the `utils/shared` modules to work even in an external module.